gemfirexd/core/src/main/javacc/com/pivotal/gemfirexd/internal/impl/sql/compile/sqlgrammar.jj

```diff
@@ -3156,7 +3156,7 @@ TOKEN :
 
 TOKEN :
 {	/* Identifiers */
-	<IDENTIFIER: ( <LETTER> ) (<LETTER> | "_" | <DIGIT>)* >
+	<IDENTIFIER: (<UNDERSCORE>)* ( <LETTER> ) (<LETTER> | "_" | <DIGIT>)* >
```

Comment:
Can you figure why it uses "_" instead of <UNDERSCORE> there? Perhaps that can also be made <UNDERSCORE> for consistency.